### PR TITLE
Add --jit flag to Benchmarker

### PR DESCRIPTION
**Pull request by GorogPeter**

### CODE REVIEW

**Comment by clover2123** [comment]
Please rebase this patch based on the main branch

File: test/wasmBenchmarker/benchmark.py [hunk 1]
```diff
@@ -143,14 +144,16 @@ def compile_tests(emcc_path, path, only_game=False, compile_anyway=False, simd=F
...
-def run_wasm(engine, path, name):
+def run_wasm(engine, path, name, jit=False):
     if not os.path.exists(path):
         print("invalid path for run")
         exit(1)
 
     tc_path = path + "/wasm/" + name + ".wasm"
     print("TC path: " + tc_path)
-    result = subprocess.check_output(engine + " --run-export runtime " + tc_path, shell=True)
+    flags = "--jit --run-export runtime" if jit else "--run-export runtime"
```
Comment (zherczeg) [inline test/wasmBenchmarker/benchmark.py:154]: This is a string duplication. Maybe this work:
`"--run-export runtime" + (" --jit" if jit else "")`

**Review by zherczeg — CHANGES_REQUESTED** [review]

File: test/wasmBenchmarker/benchmark.py [hunk 2]
```diff
@@ -161,13 +164,13 @@ def run_wasm(engine, path, name):
     return True
 
 
-def measure_time(path, name, functon, engine=None):
+def measure_time(path, name, functon, engine=None, jit=False):
```
Comment (zherczeg) [inline test/wasmBenchmarker/benchmark.py:167]: This is not introduced here, but `functon` should be changed to `function`. Otherwise the patch ok.

**Review by zherczeg — APPROVED** [review]
LGTM

**Review by clover2123 — APPROVED** [review]

**Pull request merged**